actor-tests/src/test/scala/org/apache/pekko/io/dns/DockerBindDnsService.scala

```diff
@@ -18,89 +18,143 @@ import scala.util.Try
 import scala.util.control.NonFatal
 
 import com.typesafe.config.Config
-import com.spotify.docker.client.DefaultDockerClient
-import com.spotify.docker.client.DockerClient.{ ListContainersParam, LogsParam }
-import com.spotify.docker.client.messages.{ ContainerConfig, HostConfig, PortBinding }
+import com.github.dockerjava.core.DockerClientConfig
+import com.github.dockerjava.api.model.HostConfig;
+import com.github.dockerjava.core.DefaultDockerClientConfig
+import com.github.dockerjava.api.model.Frame;
+import com.github.dockerjava.api.async.ResultCallback;
 import org.scalatest.concurrent.Eventually
 
 import org.apache.pekko
 import pekko.testkit.PekkoSpec
 import pekko.util.ccompat.JavaConverters._
+import com.github.dockerjava.api.DockerClient
+import com.github.dockerjava.httpclient5.ApacheDockerHttpClient
+import com.github.dockerjava.api.command.CreateContainerCmd
+import com.github.dockerjava.api.model.PortBinding
+import com.github.dockerjava.api.model.Container
+import com.github.dockerjava.api.model.Volume
+import com.github.dockerjava.core.DockerClientImpl
+import com.github.dockerjava.api.model.Bind
 
 abstract class DockerBindDnsService(config: Config) extends PekkoSpec(config) with Eventually {
-  val client = DefaultDockerClient.fromEnv().build()
+
+  val dockerConfig: DockerClientConfig = DefaultDockerClientConfig.createDefaultConfigBuilder().build();
```

Comment:
`DefaultDockerClient.fromEnv()` -> `DefaultDockerClientConfig.createDefaultConfigBuilder()`
This change seems to bring some changes to the configuration method, it can't configration by env.
```shell
DockerClientConfig custom = DefaultDockerClientConfig.createDefaultConfigBuilder()
    .withDockerHost("tcp://docker.somewhere.tld:2376")
    .withDockerTlsVerify(true)
    .withDockerCertPath("/home/user/.docker")
    .withRegistryUsername(registryUser)
    .withRegistryPassword(registryPass)
    .withRegistryEmail(registryMail)
    .withRegistryUrl(registryUrl)
    .build();
```
for testContainer, we can configration by env
https://java.testcontainers.org/supported_docker_environment/
please consider the question: `How can I customize the Docker configuration`,
maybe docker-java  supportes it allready.

Reply:
@laglangyue from my understanding the `createDefaultConfigBuilder` fetch the configuration from env, unless overwritten, see: https://github.com/docker-java/docker-java/blob/main/docs/getting_started.md#system-environment

Please let me know if I am missing something

Reply:
fine,it's ok

Reply:
Maybe it is necessary to add a document or a link to tell users how to configure it.